### PR TITLE
Mh c2 engineer

### PR DESCRIPTION
**Pull request by bleehu**
We've got an engineer prepped to play combat 2.0 on Saturday, so some urgency would be appreciated.

### CODE REVIEW

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt [hunk 1]
```diff
@@ -210,10 +210,10 @@ Duration: 	engagement
 Range:		Touch
 
     You fiddle with your ally's gun quickly. You tune the whosits, and
-dial the whatsits. It now has a -5% chance to miss and has half the chance of
-jamming. Normal use and wear on the gun will almost definitely unalign what 
-you've done by the end of the engagement, but hopefully your efforts will help 
-keep your ally alive until then.
+dial the whatsits. It now has a -1 chance to miss and has half the chance of
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Engineer Processes.txt:225]: Probably should reword this to something like "It loses 1 miss chance" or "It gains -1 miss chance". 
It wasn't clear to me if you meant for the gun's miss chance to become -1 or whether the gun's miss chance is reduced by -1. I assume you meant the less-powerful version.

Non-Blocking

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt [hunk 2]
```diff
@@ -559,16 +559,15 @@ Size:				1m x .6m x .8m
...
-take a -20% to miss for each other friendly PD-5 within 6m due to their 
-distributed network architecture, but if one PD bot is pwned, all PD bots within
-that range are pwned as well. Disrupted PD-5 bots don't count for other's 
-accuracy bonus.
+    This small bipedal drone comes equipped with a distributed computing NIC 
+card for improved performance in large packs. The PD-5 also comes standard with 
+a biometric sharp grasping claws and a tail that adapts procedurally to maintain 
+balance at high speeds as  well as significantly more capacity for peripherals 
+than its flying brethren. The Grasping claws may make melee attacks with a bonus
+of 4 for 40 damage. PD-5 bots take a -2 to miss for each other friendly PD-5
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Engineer Processes.txt:581]: >bonus of 4 for 40 damage

4 what? Bonus of +4 melee accuracy modifier?

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt [hunk 3]
```diff
@@ -644,9 +656,10 @@ Pwn DC:				70
 
     The titan multipurpose drone is a bipedal megalith of death
 and science. Plasma induction core explodes violently when destroyed. 
-The titan comes with a 7.62 repeater that has a 20% chance to miss and deals 55 
-damage. fed from a 150 round drum. May also make melee attacks with its offhand.
-20% chance to miss. Deals 160 damage, kinda like being punched by a backhoe.
+The titan comes with a 7.62 repeater that has a 2 chance to miss and deals 55 
+damage and has an armor piercing level of 4. Fed from a 150 round drum. May also 
+make melee attacks with its offhand with a +3 bonus. Deals 160 damage, kinda
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Engineer Processes.txt]: > +3 bonus.

+3 melee accuracy bonus?

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt [hunk 4]
```diff
@@ -644,9 +656,10 @@ Pwn DC:				70
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Engineer Processes.txt:671]: in the Titan's stats,

> Roaming range:		20m from network mote.

Network 'mote' <- is that correct?

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt [hunk 5]
```diff
@@ -847,9 +860,9 @@ Size:		1m x 1m x 1m
...
     You piece together a heavy turret with a 200 round 5.56 caliber belt
 of ammo. The ballistic plate in front offers partial cover for whomever uses it.
-Each hit deals 53 damage and the gun has a 30% chance to miss. It cannot be 
+Each hit deals 53 damage and the gun has a 3 chance to miss. It cannot be 
 aimed, but it can be ripped off of its emplacement by a character that has 8 
-strength or more. When ripped off, the gun has an additional 10% chance to 
+strength or more. When ripped off, the gun has an additional +1 chance to 
 miss. If the gun is fired four times, with four consecutive actions, the
 shooter gets a free fifth shot. Every hit that an ally (not you) makes with a
 turret that you placed gives you 30 Energy. Supporting Fire weapon.
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Engineer Processes.txt]: > Supporting Fire weapon.

I believe I removed Supporting Fire. But the turret can certainly be used to Suppress.

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt [hunk 6]
```diff
@@ -1044,7 +1057,7 @@ Repair DC of 100 to purge a pwned sentry. Sentry comes with 20 rockets. Rocket
 turret tracks with visible spectrum unless a friendly IR is being shined. In 
 which case, the turret can only see what is designated.
 
-Accuracy:	50%
+Accuracy:	5 to miss
 Damage:		80 points
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Engineer Processes.txt]: >Damage:		80 points

Probably should be 80 Explosive? I typically assume untyped damage is ballistic.

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt [hunk 7]
```diff
@@ -896,7 +909,7 @@ Health:		120
 Size:		1m x 1m x 2m
 
     You set up a machinegun turret that aims and fires on its own. The 
-turret has a 28% chance to miss and a range of 40m. Each hit deals 40 damage. 
+turret has a chance to miss of 3 and a range of 40m. Each hit deals 40 damage. 
 The sentry is vulnerable to firmware security breach at DC 60. The turret can
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Engineer Processes.txt]: No AP level?

**Review by 1sourcecontrol — APPROVED** [review]
In the essence of time, you can merge., I believe nothing is particularly blocking. @bleehu are you going to have players play from RC?

I may let Dev get merged to RC, but I won't let it get merged to Play until you (or someone else) combs through this file again. 
In areas you didn't change, I see references to "Supporting Fire", damage that is probably Explosive but not typed as Explosive, many ballistic weapons without AP levels, no references to what type of Suppression they are able to output, and other misc. descriptions that need to be updated to C2.0. 

Oh, and by the way, my rule of thumb for Suppression is "If the weapon shoots once per action*, Light suppression. Burst/Auto gets Medium Suppr. Machineguns or heavier weapons get Heavy Suppr. Unless otherwise noted on a weapon"
*Once per action without Feats or abilities. Although there is precedence for an ability granting better suppression. That'd be cool

I try to define suppression in groups, as you've probably seen in the weapons docs, to save file length.

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt (hunk 1 above)
Comment (bleehu) [reply CharacterCreation/Class-Specific Documentation/Engineer Processes.txt:225]: grammar will be improved.

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt (hunk 2 above)
Comment (bleehu) [reply CharacterCreation/Class-Specific Documentation/Engineer Processes.txt:581]: correct.

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt (hunk 4 above)
Comment (bleehu) [reply CharacterCreation/Class-Specific Documentation/Engineer Processes.txt:671]: correct. See network motes at top of document.

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt (hunk 3 above)
Comment (bleehu) [reply CharacterCreation/Class-Specific Documentation/Engineer Processes.txt]: Correct.

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt (hunk 5 above)
Comment (bleehu) [reply CharacterCreation/Class-Specific Documentation/Engineer Processes.txt]: Will update.

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt (hunk 7 above)
Comment (bleehu) [reply CharacterCreation/Class-Specific Documentation/Engineer Processes.txt]: AP level will be added.

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt (hunk 6 above)
Comment (bleehu) [reply CharacterCreation/Class-Specific Documentation/Engineer Processes.txt]: Is explosive damage a type now? How does it act?

**Comment by bleehu** [comment]
I should have merged when you gave the clear, but decided to address some of the issues while they were on the table. What do you think now?

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt (hunk 2 above)
Comment (1sourcecontrol) [reply CharacterCreation/Class-Specific Documentation/Engineer Processes.txt:581]: Meaning, can you please define it like "+4 melee accuracy modifier", then?

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt (hunk 4 above)
Comment (1sourcecontrol) [reply CharacterCreation/Class-Specific Documentation/Engineer Processes.txt:671]: kk

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt (hunk 6 above)
Comment (1sourcecontrol) [reply CharacterCreation/Class-Specific Documentation/Engineer Processes.txt]: Yes, here's a link to dev:https://github.com/bleehu/Compound_X/blob/2ae7f84809d8801510ff8a2a86a6ab915224b077/01_01_Basic_Rules.txt#L1205

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt (hunk 2 above)
Comment (1sourcecontrol) [reply CharacterCreation/Class-Specific Documentation/Engineer Processes.txt:581]: Or however you want to phrase it.. but more specifically than it is currently phrased, please.

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt (hunk 4 above)
Comment (1sourcecontrol) [reply CharacterCreation/Class-Specific Documentation/Engineer Processes.txt:671]: Sorry for missing that

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt (hunk 1 above)
Comment (1sourcecontrol) [reply CharacterCreation/Class-Specific Documentation/Engineer Processes.txt:225]: Thanks! :)

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt (hunk 2 above)
Comment (1sourcecontrol) [reply CharacterCreation/Class-Specific Documentation/Engineer Processes.txt:581]: And can you also do the one that I mentioned, below?

Comment (bleehu) [reply CharacterCreation/Class-Specific Documentation/Engineer Processes.txt:581]: Will do in a subsequent PR. For expedience, I'd like to merge this now.

**Pull request merged**